app/services/paypal_service.rb

```diff
@@ -1,7 +1,7 @@
 require 'paypal-sdk-rest'
 
 PayPal::SDK::REST.set_config(
-  mode: 'sandbox', # "sandbox" or "live"
+  mode: Rails.env.production? ? 'live' : 'sandbox',
```

Comment:
one thing to note here is that I don't think we should switch to live based on the production mode of the rails app - sometimes we might run production locally, or we might be on staging or develop servers (which run in production mode) and we still want to use the sandbox.

maybe better to have an ENV var that can be set explicitly so that the default in any mode is sandbox, but so we can set PAYPAL_SDK_MODE to 'live' on the production server and only there